README.md

```diff
@@ -1,4 +1,10 @@
 # Clean Admin Menu - Magento 2 Extension 
+
+[![Latest Stable Version](https://img.shields.io/packagist/v/redchamps/module-clean-admin-menu.svg?style=flat-square)](https://packagist.org/packages/redchamps/module-clean-admin-menu)
+[![License](https://img.shields.io/github/license/redchamps/clean-admin-menu.svg?style=flat-square)](./LICENSE) 
```

Comment:
License badge shows "Not Specified", please see the preview here https://github.com/redchamps/clean-admin-menu/blob/57d61a20f0ebedf58fb064dc9bd02ac53452d428/README.md

Reply:
Ho yes right, I don't know the module license so I didn't set anything. I would suggest MIT, but it's up to you :)

Reply:
Yes, let's use MIT.

Reply:
Thanks for adding license-related changes. However, It still shows "Not Specified" in the badge. Seems like we need to use code like here https://gist.github.com/lukas-h/2a5d00690736b4c3a7ba#the-mit-license for the badge

Reply:
ahh.. my bad. Seems like it will start showing MIT as soon as the pull request is merged. Because the shield URL https://img.shields.io/github/license/redchamps/clean-admin-menu.svg contains username "redchamps" instead of "thomas-kl1"

I am merging the pull request now.